src/iidm/converter/xml/TerminalRefXml.cpp

```diff
@@ -57,6 +59,11 @@ void TerminalRefXml::writeTerminalRef(const Terminal& terminal, NetworkXmlWriter
     if (!context.getFilter().test(c)) {
         throw PowsyblException(stdcxx::format("Oups, terminal ref point to a filtered equipment %1%", c.get().getId()));
     }
+    if (terminal.getVoltageLevel().getTopologyKind() == TopologyKind::NODE_BREAKER
+        && context.getOptions().getTopologyLevel() != TopologyLevel::NODE_BREAKER
+        && stdcxx::isInstanceOf<BusbarSection>(terminal.getConnectable())) {
```

Comment:
I would put the `&&` operators at the end of the line instead of the beginning of the lines